ls1intum/Artemis
# Add all students from course to exam

### Checklist

### Motivation and Context
Currently adding multiple students to an exam can only be done via a csv file. Registering all students who are enrolled in the course is useful when creating exams. 

### Description

We add a Button the Instructor Page 'exam-students', which instructors can use the register all students to the exam which are currently enrolled in the course.  (E.g. the student has the courses student-group assigned)

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration -> Course -> Exam -> Students
3. Clicking on the new Button should register all students to the exam, who are currently enrolled in this course. 
4. This will enroll all users into the exam who are part of the courses student-group! 
5. No other students get registered to the exam. 
6. **Instructors of this course** or **Admins** will not be added to the exam! 

### Test Coverage




ExamResourse: 98.5%
exam-students-component: 79%
exam-managment-component: 85.7%
exam-managment-service: 81%



### Screenshots


![Bildschirmfoto 2021-01-30 um 13 49 29](https://user-images.githubusercontent.com/42205689/106356750-48f48180-6302-11eb-9ee1-181e7d04ad8c.png)
